### ISSUE TITLE
Make available as static binary

### ISSUE THREAD

**Issue by yeroc** [protocolbuffers/protoscope#23]
Is it possible to make protoscope available as a binary for the common platforms (at least Linux)?  For anyone not using go as their language it's rather unfriendly to require the whole language toolchain etc. to install.

Thanks!

**Comment by 3052** [comment]
> For anyone not using go as their language it's rather unfriendly to require the whole language toolchain etc. to install.

not really. if you replaced "go" with some old crappy language sure, but the download is only 68 MB:

~~~
> curl -I https://dl.google.com/go/go1.22.2.linux-amd64.tar.gz
HTTP/2 200
content-length: 68958123
~~~

and building is literally `go build`, thats it. no other downloads, no other commands, nothing.
